generatebundlefile/hack/release_prod.sh

```diff
@@ -84,8 +84,15 @@ function push () {
     local version=${1?:no version specified}
     cd "${BASE_DIRECTORY}/generatebundlefile/output-${version}"
     awsAuth "$REPO" | "$ORAS_BIN" login "$REPO" --username AWS --password-stdin
-    "$ORAS_BIN" push "${REPO}:v${version}-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-    "$ORAS_BIN" push "${REPO}:v${version}-latest" bundle.yaml
+    "$ORAS_BIN" pull "${REPO}:v${version}-latest" -o ${version}
+    removeBundleMetadata ${version}/bundle.yaml
+    removeBundleMetadata bundle.yaml
+    if (git diff --no-index --quiet -- ${version}/bundle.yaml.stripped bundle.yaml.stripped) then
```

Comment:
technically, should be no need to use `git diff` over `diff` here and it should not require parens on this